dataactcore/models/stagingModels.py

```diff
@@ -522,8 +522,10 @@ class DetachedAwardFinancialAssistance(Base):
     awardee_or_recipient_legal = Column(Text)
     awardee_or_recipient_uniqu = Column(Text)
     awarding_agency_code = Column(Text)
+    awarding_agency_name = Column(Text)
     awarding_office_code = Column(Text)
     awarding_sub_tier_agency_c = Column(Text)
+    awarding_sub_tier_agency_n = Column(Text)
```

Comment:
These shouldn't be adding to the detached one, we aren't asking them to include the title in their file, we're deriving it on submit